master/docs/manual/cfg-properties.rst

```diff
@@ -234,9 +235,28 @@ The following ways of interpreting the value are available.
 Example::
 
     from buildbot.plugins import steps, util
-    f.addStep(steps.ShellCommand(command=['make',
-                                          util.Interpolate('REVISION=%(prop:got_revision:-%(src::revision:-unknown)s)s'),
-                                          'dist']))
+    f.addStep(steps.ShellCommand(
+        command=[
+            'save-build-artifacts-script.sh',
+            util.Interpolate('-r %(prop:repository)s'),
+            util.Interpolate('-b %(src::branch)s'),
+            util.Interpolate('-d %(kw:data)s', data="some extra needed data")
+        ]))
+
+.. note::
+
+   We use ``%(src::branch)s`` in most of examples, because ``codebase`` is empty by default.
+
+
+Example::
```

Comment:
please use explicit `.. code-block: python`, we had issues when automatic detection of the language did not work (while you are at it, please apply the same change to `Example::` above).